src/assessor_tools.yml

```
@@ -389,21 +403,69 @@
     - name: Install system packages for tools where possible
       ansible.builtin.package:
         name:
+          - assetfinder
+          - cargo
           - cifs-utils
           - dnsutils
           - exploitdb
           - eyewitness
+          - feroxbuster
           - firefox-developer-edition-en-us-kbx
           - flameshot
+          - getallurls
           - gobuster
+          - golang
+          - jq
           - libreoffice
           - nfs-common
           - nikto
+          - nmap
           - powersploit
+          - subfinder
           - sublist3r
           - terminator
           - testssl.sh
           - veil
+          - vim
+
+    # In order for the VNC user to make full use of Go, we need to add
+    # $p/bin to the PATH for all paths p in $GOPATH.
+    #
+    # See #137 for more details.
+    - name: Add GOPATH binary paths to PATH for VNC user
+      ansible.builtin.blockinfile:
+        # The jinja2 comment above changes the Jinja2 templating to
```

Unless I've misunderstood something.

```suggestion
        # The jinja2 comment below changes the Jinja2 templating to
```